src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/GEOS_LandAssimGridComp.F90

```diff
@@ -1277,26 +1344,26 @@ subroutine Initialize(gc, import, export, clock, rc)
     call MPI_BCAST(N_obsbias_max,         1, MPI_INTEGER,        0,MPICOMM,mpierr)
    
 
-    ! create LandAssimAlarm with option of centered update 
+    ! create LandAssimAlarm
 
-    if (centered_update)then
-       rrTime = CurrentTime+Landassim_DT/2-ModelTimeStep 
-    else
-       rrTime = CurrentTime+Landassim_DT-ModelTimeStep 
-    endif
+    ! RR20210506: If AssimTime is indeed the time of the first land analysis, this should be "rrTime=AssimTime-ModelTimeStep"
+
+    !  NOT SURE THE ABOVE COMMENT MADE SENSE AT THE TIME OR NOW. NEED TO DOUBLE-CHECK AND CLEAN UP.
+    
+    rrTime = AssimTime - ModelTimeStep        ! probably rrTime=AssimTime is correct????
```

Comment:
@weiyuan-jiang, I'm not sure this is correct.  Shouldn't it be rrtime=AssimTime after the recent revisions?

Reply:
The CurrentTime is not updated until HistoryGC run().  So when is time to do data assimilation, the time in LandAssimGc is still AssimTim -  ModelTimeStep and it should ring at that time

Reply:
Fixed with most recent commit.